### PR TITLE
Conditional autoregressive prior example notebook

### PR DESCRIPTION
**Pull request by conorhassan**
Here is a new notebook using `pm.CAR` distribution. The linked issue is #405.

The notebook uses parts of the code from the already existing [CAR notebook example](https://docs.pymc.io/en/v3/pymc-examples/examples/case_studies/conditional-autoregressive-model.html), but is updated to v4, uses `pm.CAR` and has a few other bits and pieces.

*Checklist:*
+ [x] Notebook follows style guide https://docs.pymc.io/en/latest/contributing/jupyter_style.html
+ [x] PR description contains a link to the relevant issue: a tracker one for existing notebooks or a proposal one for new notebooks
+ [ ] Check the notebook is not excluded from any pre-commit check: https://github.com/pymc-devs/pymc-examples/blob/main/.pre-commit-config.yaml

I am getting jupytext pre-commit errors and am unsure how to fix them.

### CODE REVIEW

File: examples/references.bib [hunk 1]
```diff
@@ -278,6 +296,20 @@ @misc{kucukelbir2015automatic
   archiveprefix = {arXiv},
   primaryclass  = {stat.ML}
 }
+@misc{lao2020conditional,
+  author        = {Lao, Junpeng},
+  title         = {Conditional Autoregressive (CAR) model},
+  url           = {https://docs.pymc.io/en/v3/pymc-examples/examples/case\_studies/conditional-autoregressive-model.html},
+  year          = {2020}
+}
```
Comment (OriolAbril) [inline examples/references.bib:304]: This one should not be a bibtex reference but a sphinx cross reference.

File: myst_nbs/case_studies/conditional_autoregressive_priors.myst.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+substitutions:
+  extra_dependencies: bambi seaborn
+---
+
+(conditional_autoregressive_priors)=
+# Conditional Autoregressive (CAR) Models for Spatial Data
+
+:::{post} Jul 29, 2022 
+:tags: beginner, tutorial, spatial
+:category: beginner
```
Comment (OriolAbril) [inline myst_nbs/case_studies/conditional_autoregressive_priors.myst.md:21]:
```suggestion
:tags: spatial
:category: beginner, tutorial
```

beginner and tutorial should be categories and not tags.

File: myst_nbs/case_studies/conditional_autoregressive_priors.myst.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+
+:::{include} ../extra_installs.md
+:::
+
+```{code-cell} ipython3
+# import mapclassify
+import libpysal
+
+# import bokeh
+from bokeh.plotting import output_notebook
```
Comment (OriolAbril) [inline myst_nbs/case_studies/conditional_autoregressive_priors.myst.md:42]: I think these two are not dependencies of pymc either.

File: myst_nbs/case_studies/conditional_autoregressive_priors.myst.md [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3.10.5 ('pymc-dev-car-nb')
+  language: python
+  name: python3
+substitutions:
+  extra_dependencies: bambi seaborn
```
Comment (OriolAbril) [inline myst_nbs/case_studies/conditional_autoregressive_priors.myst.md:13]: It doesn't look like bambi is used. The libraries here are the ones that should be installed in addition to pip install pymc.

**Review by OriolAbril — COMMENTED** [review]
had to stop not long after starting

**Comment by bwengals** [comment]
Thanks for reviewing @OriolAbril -- Yup you're right there's a ways to go.  Hey @conorhassan do you mind [converting it to draft?](https://stackoverflow.com/questions/55070723/how-do-i-change-from-a-pr-to-a-draft-pr-at-github).  That's my bad I forgot to mention that yesterday.  If it's in draft it pretty much just signals "no intent to merge yet, just putting this up publicly so it's visible and any feedback is welcome".  Not in draft signals "Pretty much ready, any last changes that I could/should make before merging this?"

Quick Q though @OriolAbril, if you don't mind.  This is a follow up from #405.  Do you know why pre-commit is failing?  We fixed the `ther` issue, but something is still amiss.  We didn't have much luck yesterday.

We tried commiting the changes seen in the current diff made by jupytext, but iirc something else fails, we regenerate the myst file, and we ended up here again.

If it's not super obvious, I'll be able to check out the PR and give it another try in a few days.

**Comment by OriolAbril** [comment]
Just saw I missed the notification for this PR. I don't know why it is complaining. I would try rebasing and seeing how it goes. Otherwise I can try to check out this PR and take a look

**Comment by bwengals** [comment]
Is pre-commit correcting some spelling issues?  What I see in pre-commit isnt in the PR diff

**Comment by OriolAbril** [comment]
> Is pre-commit correcting some spelling issues?

yes, one of the pre-commit jobs is a spellcheck

**Comment by daniel-saunders-phil** [comment]
I just wanted to say I'd like to pick the work on this and make sure this gets merged. I'll dig in tomorrow morning and hopefully send a PR then.

**Comment by daniel-saunders-phil** [comment]
#547 replaces this pr - should be safe to close at any point.

**Pull request closed without merging**